.github/workflows/actions/publish-npm/action.yml

```diff
@@ -17,19 +17,23 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
+    - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
+      with:
+        version: 8
+    - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
       with:
         node-version-file: './package.json'
+        cache: 'pnpm'
     - name: Install Dependencies
-      run: npx lerna bootstrap --include-dependencies --scope ${{ inputs.scope }} --ignore-scripts -- --legacy-peer-deps
+      run: pnpm --filter ${{ inputs.scope }} install
       shell: bash
       working-directory: ${{ inputs.working-directory }}
     - name: Update Version
       run: npx lerna version ${{ inputs.version }} --yes --exact --no-changelog --no-git-tag-version --preid=${{ inputs.preid }}
```

Comment:
Personally I have not used `pnpm version` or was aware if it versions similar to lerna or falls back to it. If the team is interested in that, that is something that could continued to be investigated in further work. To leave as much of the infrastructure in place I would say keeping lerna provides confidence that versioning and the dev & prod release workflows continue to operate as they do today (and similar to other projects the team maintains, i.e.: Ionic Framework).

Reply:
That's fine - I was just trying to understand what lerna did/brought to the table here.  Unfortunately, I think the pnpm/lerna knowledge gap is one the Framework + Stencil teams will have to bridge (moreso the Stencil team with respect to lerna, as that team doesn't use lerna in any of its projects) - with that in mind, I was trying (and I suppose my goal still is) to get a baseline for why we do certain things the way we do today, since figuring them out after some period of time is often more trying in my experience 😉